src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -14,40 +13,29 @@ namespace Azure.ApplicationModel.Configuration
 {
     public partial class ConfigurationClient
     {
-        static readonly HttpPipelinePolicy s_defaultRetryPolicy = RetryPolicy.CreateFixed(3, TimeSpan.Zero,
-            //429, // Too Many Requests TODO (pri 2): this needs to throttle based on x-ms-retry-after
-            500, // Internal Server Error
-            503, // Service Unavailable
-            504  // Gateway Timeout
-        );
-
         private readonly Uri _baseUri;
         private readonly HttpPipeline _pipeline;
 
-        public static HttpPipelineOptions CreateDefaultPipelineOptions()
-        {
-            var options = new HttpPipelineOptions(HttpClientTransport.Shared);
-            options.LoggingPolicy = LoggingPolicy.Shared;
-            options.RetryPolicy = s_defaultRetryPolicy;
-            return options;
-        }
-
         public ConfigurationClient(string connectionString)
-            : this(connectionString, CreateDefaultPipelineOptions())
+            : this(connectionString, new ConfigurationPipelineOptions())
         {
         }
 
-        public ConfigurationClient(string connectionString, HttpPipelineOptions options)
+        public ConfigurationClient(string connectionString, ConfigurationPipelineOptions options)
```

Comment:
This type should be called ConfigurationClientOptions. It will have more than just pipeline configuration

Reply:
Do you want to move the connection string there too?

Reply:
@schaabs is working on a proposal where auth types will go there. Not sure if connection string will be one of them.